rancid-tomatillos/src/PreviewCard/PreviewCard.test.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import {render, screen} from "@testing-library/react";
+import "@testing-library/jest-dom";
+import userEvent from "@testing-library/user-event";
+import PreviewCard from "./PreviewCard";
+
+describe("Card", () => {
+  beforeEach(() => {
+    const fakeClick = { push: jest.fn(id) }
+ 
+    render(
+      <PreviewCard
+        average_rating={9}
+        backdrop_path="testback"
+        id={694919}
+        poster_path="testpath"
+        release_date="test-date"
+        title="Test Title"
+        onClick={fakeClick}
```

Comment:
Hopefully we can figure out how to get this working, route testing lesson may help us here

Reply:
Would `jest.mock()` be applicable here? Need to do more research on my end

Reply:
You may need a spy here!